engine/src/flutter/shell/common/shell.cc

```diff
@@ -1080,7 +1079,6 @@ void Shell::OnPlatformViewSetViewportMetrics(int64_t view_id,
 
   if (metrics.device_pixel_ratio <= 0 || metrics.physical_width <= 0 ||
       metrics.physical_height <= 0) {
```

Comment:
Could we add a check that the size is within the constraints?

Reply:
Added.